core/Updater.php

```diff
@@ -279,6 +279,15 @@ public function update($componentName)
 
         $this->executeListenerHook('onComponentUpdateFinished', array($componentName, $updatedVersion, $warningMessages));
 
+        /**
+         * Event triggered after a component has been updated.
+         *
+         * @param string $componentName 'core', or plugin name
+         * @param string $updatedVersion version updated to
+         * @param array  $warningMessages warnings occurred during update
+         */
+        Piwik::postEvent('Updater.componentUpdated', array($componentName, $updatedVersion, $warningMessages));
```

Comment:
An example on how to use it might be useful. Also an example maybe for `$updatedVersion` and `$warningMessages` could be helpful. I wonder if we could refactor the `$this->executeListenerHook('onComponentUpdateFinished', array($componentName, $updatedVersion, $warningMessages));` to use this event now. would make things simpler to have only one of them